src/posit/connect/permissions.py

```diff
@@ -67,36 +67,105 @@ def count(self) -> int:
         return len(self.find())
 
     @overload
-    def create(self, *, principal_guid: str, principal_type: str, role: str) -> Permission:
+    def create(self, *, principal_guid: str, principal_type: str, role: str) -> Permission: ...
+
+    @overload
+    def create(self, *args: User | Group, role: str) -> list[Permission]: ...
+
+    def create(self, *args: User | Group, **kwargs) -> Permission | list[Permission]:
         """Create a permission.
 
         Parameters
         ----------
+        *args : User | Group
+            The principal users or groups to add.
+        role : str
+            The principal role. Currently only `"viewer"` and `"owner"` are supported.
         principal_guid : str
+            User guid or Group guid.
         principal_type : str
+            The principal type. Either `"user"` or `"group"`.
         role : str
+            The principal role. Currently only `"viewer"` and `"owner"` are supported
 
         Returns
         -------
-        Permission
-        """
-
-    @overload
-    def create(self, **kwargs) -> Permission:
-        """Create a permission.
+        Permission | List[Permission]
+            Returns a `Permission` when the kwargs: `principal_guid` and `principal_type` are used.
+            Returns a `list[Permission]` when `*args` are used.
 
-        Returns
-        -------
-        Permission
-        """
+        Examples
+        --------
+        ```python
+        from posit import connect
 
-    def create(self, **kwargs) -> Permission:
-        """Create a permission.
+        client = connect.Client()
+        content_item = client.content.get(content_guid)
+
+        # New permission role
+        role = "viewer"  # Or "owner"
+
+        # Example groups and users
+        groups = client.groups.find(prefix="GROUP_NAME_PREFIX_HERE")
+        user = client.users.get("USER_GUID_HERE")
+        users = [user]
+
+        # Add many group and user permissions with the same role
+        content_item.permissions.create(*groups, *users, role=role)
```

Comment:
If you make the above change, you could replace this with an example using a list comprehension on a list of groups and/or users.